slides/src/content/bios/philip-dasilva.md

```diff
@@ -0,0 +1,11 @@
+## Philip Da Silva
+
+<img src="content/bios/images/philip-dasilva.png" alt="Philip Da Silva" style="max-width: 200px;" />
+
+Philip Da Silva is a Solutions Architect at Rangle.io specializing in front-end
+technologies and UI and is a self-proclaimed
+*[single div CSS artist](http://codepen.io/collection/DqBbZb/)*.
```

Comment:
Following convention for the links you added in your own - I don't need the italics, shall I remove them?